lib/screens/chat_screen.dart

```diff
@@ -29,11 +31,57 @@ class _ChatScreenState extends State<ChatScreen> {
 
   @override
   Widget build(BuildContext context) {
-    return Container(
-      child: SafeArea(
-        //TODO Implement Chat Screen
-        child: Text('Chat screen'),
-      ),
-    );
+    MediaQueryData queryData;
+    queryData = MediaQuery.of(context);
+    return Scaffold(
+        backgroundColor: ChatScreenBackground,
+        appBar: AppBar(
+          backgroundColor: ChatScreenAppBar,
+        ),
+        body: SafeArea(
+            child: Column(
+          mainAxisAlignment: MainAxisAlignment.end,
+          crossAxisAlignment: CrossAxisAlignment.start,
+          children: [
+            Padding(
+                padding: const EdgeInsets.all(10.0),
+                child: Row(children: [
+                  Padding(
+                    padding: const EdgeInsets.all(12.0),
+                    child: Align(
+                      alignment: Alignment.bottomLeft,
+                      child: TextButton(
+                        onPressed: () {},
+                        style: TextButton.styleFrom(
+                          backgroundColor: Colors.white,
+                          shape: RoundedRectangleBorder(
+                            side: BorderSide(color: ChatButtonBorder, width: 2),
+                            borderRadius: BorderRadius.circular(50.0),
+                          ),
+                          elevation: 1.0,
+                          minimumSize: Size(queryData.size.width * 0.7,
+                              queryData.size.height * 0.06),
+                        ),
+                        child: Text(
+                          "",
+                          style: TextStyle(color: Colors.black),
+                        ),
+                      ),
```

Comment:
Because we need to write text in this section, please use `TextField` widget rather than `TextButton`.